### ISSUE TITLE
Why the loss is identically one?

### ISSUE THREAD

**Issue by keithyh** [WenmuZhou/PSENet.pytorch#61]
Dear author,thank you for your great job.I trained my own data according to your instructions.

During training,I found the loss is always 1 as shown in the picture below.
![loss1](https://user-images.githubusercontent.com/14086114/74314412-5989b000-4db0-11ea-9e63-8f492fbfe0f6.png)

Could you tell me the possible reason leading to this error?

**Comment by Kennycao123** [comment]
I have the same problem,do you have solve it?

**Comment by WenmuZhou** [comment]
label of text must not be # and ***

**Comment by Kennycao123** [comment]
可否加个老哥微信聊，我微信(18022479858)


 
---Original---
